css/about.css

```diff
@@ -0,0 +1,442 @@
+  background-image: url(../img/Desktop/last_posts.jpg);
+  background-size: 540px 300px;
+  background-repeat: no-repeat;
+  text-decoration: none;
+}
+
+.See_post .post1,
+.post2 {
+  width: 540px;
+  height: 300px;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-bottom: 50px;
+}
+
+.post2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url(../img/Desktop/last_post2.jpg);
+  background-size: 540px 300px;
+  background-repeat: no-repeat;
+  text-decoration: none;
+}
+
+.post1:hover {
+  background-image: url(../img/Desktop/posts_hover.jpg);
+}
+
+.post2:hover {
+  background-image: url(../img/Desktop/post2_hover.jpg);
+}
+
+.post1 h3,
+.post2 h3 {
+  background: rgba(255, 255, 255, 0.1);
+  text-align: center;
+  color: #ec5242;
+  font-size: 48px;
+  font-family: "lato", sans-serif;
+}
+
+footer {
+  max-width: 1100px;
+  margin: 0 auto;
+  height: 260px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.mobile-menu {
+  display: none;
+}
+
+@media only screen and (max-width: 768px) {
+  header {
+    display: none;
+  }
+
+  nav {
+    display: none;
+  }
+
+  main .main-about .hellotxt h1 {
+    font-size: 30px;
+    line-height: 51px;
+    letter-spacing: 2px;
+  }
+
+  .description {
+    height: 200px;
+    width: calc(85% - 40px);
+    margin-left: 10px;
+  }
+
+  section > div > p {
+    padding: 0 35px 50px 35px;
+  }
+
+  .description p {
+    font-size: 17px;
+    font-weight: 700;
+  }
+
+  main .main-about .main-other-des {
+    font-size: 22px;
+  }
+
+  section > div > h3 {
+    font-size: 29px;
+    text-align: center;
+  }
+
+  section > div > img {
+    height: 160px;
+    width: 330px;
+  }
+
+  .See_post {
+    flex-direction: column;
+  }
+
+  .post1,
+  .post2 {
+    background-size: 315px 185px;
+  }
+
+  .See_post .post1,
+  .post2 {
+    width: 315px;
+    height: 185px;
+    padding-bottom: 20px;
+  }
+
+  .post1 h3,
+  .post2 h3 {
+    font-size: 35px;
+  }
+
+  .post1 p,
+  .post2 p {
+    font-size: 18px;
+  }
+
+  .mobile-menu {
+    display: block !important;
+    position: absolute;
+  }
+
+  .material-icons {
+    font-size: 64px !important;
+  }
+}
+
+.menuItem {
+  display: block;
+  margin: 2rem 4rem;
+  font-size: 1.8rem;
+  color: #888;
+  text-decoration: none;
+}
+
+.menuItem:hover {
+  border-bottom: #ec5242 solid 2px;
+}
+
+.hamburger {
+  position: fixed;
+  z-index: 100;
+  top: 1rem;
+  left: 1rem;
+  padding: 4px;
+  border: none;
+  color: #888;
+  background: none;
+  cursor: pointer;
+}
+
+.closeIcon {
+  display: none;
+}
+
+.menu {
+  position: fixed;
+  transform: translateY(-100%);
+  transition: transform 0.2s;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #fff;
+
+  /* color: white; */
+  list-style: none;
+  padding-top: 4rem;
+}
+
+.showMenu {
+  transform: translateY(0);
+}
+
+.material-icons {
+  font-size: 60px;
+}
```

Comment:
- [ ] There is a bug on your mobile menu hamburger icon and close icon which are both shown when we switch on mobile.
![ham-menu](https://user-images.githubusercontent.com/40503704/181839644-0a610d2a-1daf-4eed-a76b-e61e1b0ac5b1.PNG)

The reason for this is that somewhere in your CSS code both are initialized to be displayed `display: inline-block;` meanwhile, we only want the hamburger menu to be displayed first and the close icon to be hidden.

![css-ham](https://user-images.githubusercontent.com/40503704/181840313-ae261f8e-51b9-40b0-93a0-b1cc2f8e6960.PNG)